### ISSUE TITLE
Sorting on collection attribute

### ISSUE THREAD

**Issue by gmjordan** [ArangoDB-Community/arangodb-graphql-java#4]
Is sorting provided for on edge and vertex collections?

**Comment by colinfindlay** [comment]
I've pushed a commit to my fork that should handle this.

The way it should work is if you declare arguments on your query operation named,"limit", "skip", or "sort" it will handle them accordingly.

Limit and Skip have the same meaning as the AQL documentation for Limit and Skip, and should be declared as Int type. Sort can be declared as a custom input type, as long as all its properties are scalar and can represent the String "ASC" or "DESC". An enum is a good choice here I think.

Example Schema:

```
enum SortDirection {
    ASC,
    DESC
}

input ClientSort {
    firstName: SortDirection
    lastName: SortDirection
}

type Query {
    getClients(limit: Int, skip: Int, sort: ClientSort): [Client]
}
```

Example Query

```
query {
  getClients(limit: 1, skip: 1, sort: { lastName: ASC }) {
    firstName
    lastName
  }
}
```

If you think that works for your use case let me know and I will open a PR with the community repo, and update the docs.

**Comment by gmjordan** [comment]
I've tested it locally and it works well.  Nicely done.

**Comment by colinfindlay** [comment]
Functionality included in v1.2
